pkg/manifestreader/manifestloader.go

```diff
@@ -70,3 +98,7 @@ func mReader(path string, reader io.Reader, readerOptions ReaderOptions) Manifes
 	}
 	return mReader
 }
+
+type Loader interface {
```

Comment:
I think we can just keep this interface and the implementation in the same package as the status cobra command. It is only used there, so it makes sense to keep it there.

Reply:
@mortent Yeah that makes sense, I already created a Loader interface with a GetInvInfo function to the status cobra command.